cpuset/cpuset_test.go

```diff
@@ -0,0 +1,332 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
2023?

Reply:
Resolving this similar to 'cpuset.go' comment that thockin resolved on copyright preservation.